README.md

```diff
@@ -854,6 +854,7 @@ Besides libraries, there're [Collection on Codepen](http://codepen.io/collection
 * [Anime.js](http://animejs.com) - A JavaScript animation engine http://animejs.com.
 * [Mo.js](http://mojs.io) - Motion graphics toolbelt for the web http://mojs.io.
 * [particles.js](https://github.com/VincentGarreau/particles.js) - A lightweight JavaScript library for creating particles.
+* [Proton](https://github.com/a-jie/Proton) - Proton is a lightweight and powerful Javascript particle animation library. http://a-jie.github.io/Proton/
```

Comment:
Can you please remove the link at the end of description?

(I know, previous links have some, they'll be removed at some point)

Reply:
your suggestion is right